apps/condo/domains/organization/schema/OrganizationEmployeeRequest.js

```diff
@@ -247,6 +269,18 @@ const OrganizationEmployeeRequest = new GQLListSchema('OrganizationEmployeeReque
                 }
             }
 
+            const userId = get(resolvedData, 'user')
+            if (userId) {
+                const user = await getByCondition('User', {
```

Comment:
can we use getById here?

Reply:
Then you will need to manually process the case with `deletedAt: true`, I will leave it like this